pkg/extensions/customresources_test.go

```diff
@@ -699,6 +699,39 @@ var _ = Describe("extensions", func() {
 				Expect(item.Annotations[v1beta1constants.GardenerOperation]).To(Equal(v1beta1constants.GardenerOperationMigrate))
 			}
 		})
+
+		It("should properly annotate only the desired extension objects for migration", func() {
+			for i := 0; i < 4; i++ {
+				containerRuntimeExtension := &extensionsv1alpha1.ContainerRuntime{
+					ObjectMeta: metav1.ObjectMeta{
+						Namespace: namespace,
+						Name:      fmt.Sprintf("containerruntime-%d", i),
+					},
+				}
+				Expect(c.Create(ctx, containerRuntimeExtension)).To(Succeed())
```

Comment:
You can add a description in the `Succeed()` call that better indicates on which iteration of the loop a test failed. Ref https://github.com/gardener/gardener/blob/master/docs/development/testing.md#writing-tests-relevant-for-all-kinds